_posts/2020-11-08-working-with-seqdata-post.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title:  "Welcome to Working with Sequence Data!"
+date:   2020-11-08
+categories: 1. ***Overview of FASTA/FastQ***, ***Basic of counting FASTA/FASTQ entries***, ***Trimming low-quality bases***, ***Overview of Parsing FASTA/FASTQ files***
+
+---
+![seq_image](https://i.ytimg.com/vi/fCd6B5HRaZ8/maxresdefault.jpg)
+
+## **FASTA & FASTQ**
+
```

Comment:
Could you possibly add a short line about why FASTA/FASTQ sequences are important or why we should process them differently? Just a brief intro for reference in case someone stumbles upon this post by accident?

Reply:
I agree - currently, the post assumes the reader knows about fasta/fastq file formats. Since the target audience is a beginner in bioinformatics, I also suggest providing a brief summary about what fasta and fastq files are, what they contain, and what kind of information you can glean from either file format.